src/components/user/user.service.ts

```diff
@@ -117,7 +117,7 @@ export class UserService {
     return {
       items: result.items,
       hasMore: result.hasMore,
-      total: result.total,
+      total: result.items.length,
```

Comment:
The `items` is only the current page. The `total` is the total across all pages. Thus you can't use the length of the current page to determine the `total`. If this were the case we wouldn't even ask for it and just use the length of the `items` ourselves.

Reply:
I see... so this issue https://github.com/SeedCompany/cord-api-v3/issues/707 is not issue, right?

Reply:
If `hasMore` is true then `items` should not be empty. Also page 60 * count of 100 is offset 60,000 with the supposed total of 78436 again the items should not be empty.